src/main/java/org/example/JavaDatatype.java

```
@@ -0,0 +1,39 @@
+package org.example;
+
+import java.util.*;
+
+class JavaDatatype{
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Please enter count of the numbers you want to input");
+        int count=sc.nextInt();
+        System.out.println("Enter those numbers");
+        for(int i=0;i<count;i++) {
+            String n = sc.next();
+            System.out.println(findType(n));
+            System.out.println();
+        }
+    }
+    static String findType(String s) {
+        long x;
+        try{
+            x = Long.parseLong(s);
+        }catch (Exception e) {
```

think of a custom exception and instead of returning error string "s + " can't be fitted anywhere";" just throw your custom exception with that error message. In tests try to test that exception is thrown.